src/main/java/employment/system/controllers/SuccessfulApplyingController.java

```diff
@@ -0,0 +1,25 @@
+package employment.system.controllers;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class SuccessfulApplyingController {
+    public Button okButtonField;
+
+    public void okButtonOnAction(ActionEvent actionEvent) {
+        try {
+            Stage stage = (Stage) okButtonField.getScene().getWindow();
+            Parent openRegistrationTab = FXMLLoader.load(getClass().getClassLoader().getResource("login.fxml"));
+            Scene scene = new Scene(openRegistrationTab, 600, 400);
+            stage.setScene(scene);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+}
```

Comment:
Please add newline at the end of the file.

Reply:
I will do that in the next story ( ESFI-27 - Refactoring) because we decided to modify some issues there.